ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -450,8 +450,8 @@ private class JdbcLedgerDao(
               .foreach(_.execute())
           )
         } else {
-          for (info @ SubmitterInfo(submitter, _, commandId, _) <- submitterInfo) {
-            stopDeduplicatingCommandSync(domain.CommandId(commandId), submitter)
+          for (info @ SubmitterInfo(actAs, _, commandId, _) <- submitterInfo) {
```

Comment:
Is `actAs` used here?

Reply:
No, it will be used in one of the upcoming PRs. I renamed it preemptively to help the future PR that touches this line.